NDTensors/test/runtests.jl

```diff
@@ -1,5 +1,6 @@
 using Test
 using SafeTestsets
+using GPUArraysCore
```

Comment:
I think we don't need this at the top level and can just rely on importing it properly where it is needed.

Reply:
Makes sense, since its in the `device_lists.jl` file. I have removed it from the top level